resources/views/Checkout/CheckoutView.twig

```diff
@@ -20,6 +20,28 @@
     {{ component( "Ceres::Basket.Components.BasketTotals" ) }}
     {{ component( "Ceres::Checkout.Components.Checkout" ) }}
 
+    {% set optionalInvoiceAddressFields = {
```

Comment:
@cjohannsen @lks-mtzn Bitte auf einheitliche Abkürzungen einigen. Derzeit haben wir hier "gb" als Key, "_en" als Suffix und "uk" als Key in den AddressSelect-Komponenten.